pages/strategy.md

```diff
@@ -0,0 +1,58 @@
+Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy. Our current goal is to make Exercism scalable. We expect our next three goals to be to make Exercism financially sustainable, to make Exercism more accessible to beginners, and to intentionally grow Exercism for the first time.
+
+We present this as an Objective, which will aim to achieve with a Strategy, supported by a series of Tactics. This is commonly known as the OST framework.
+
+## Objective
+
+**Make Exercism scalable**
+
+At the time of writing, Exercism has 8,000 new submissions per week. Our mentors currently cannot keep up with the amount of submissions submitted. When they do keep up, the number of submissions almost immediately doubles, as students stick with a Language Track. We are growing by 300-400 new users per day. Many find Exercism too hard to use, but in the future, when that is fixed, we will expect to see the number of submissions double again. We have never actively marketed Exercism. All 300,000 members have come through word of mouth. When we start to market Exercism, we expect to see an increase in usage by two orders of magnitude. Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
```

Comment:
Same as the long read all the way at the start of the document, I would separate the concrete goal from the paragraph, like so (including Katrina's change):

```suggestion
At the time of writing, Exercism has 8,000 new submissions per week. Our mentors currently cannot keep up with the number of submissions submitted. When they do keep up, the number of submissions almost immediately doubles, as students stick with a Language Track. We are growing by 300-400 new users per day. Many find Exercism too hard to use, but in the future, when that is fixed, we will expect to see the number of submissions double again. We have never actively marketed Exercism. All 300,000 members have come through word of mouth. When we start to market Exercism, we expect to see an increase in usage by two orders of magnitude. 

Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
```

I also don't now if it's preferable to combine so many different subjects into one block. You're talking about scalability, but also that many find exercism too hard which targets the progression lines and learning curve. 

I think it might be good to remove that all together and instead write that "those track that have been undergoing restructure via the project track anatomy" have seen significant increase in the number of submissions, or something similar in your words. That said, since in the **Tactics** part you elaborate on this, I think this paragraph works fine without the entire sentence.

Reply:
For clarity, I mean "hard to use" as in things like downloading the CLI rather than finding the exercises too hard.

Reply:
Now that you say it, it makes sense. This is not clear from the wording 🗡 😅